src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,14 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        int a = value;
```

Comment:
do we need these variables? https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html